etcd/getting-started-with-etcd.md

```diff
@@ -6,6 +6,47 @@ Application containers running on your cluster can read and write data into etcd
 
 <a class="btn btn-default" href="../latest/api.html">Complete etcd API Docs</a>
 
+## Setting up etcd
+
+Container Linux's `etcd-member.service` systemd unit knows how to fetch and run the current etcd v3.x container image, providing etcd v3 without requiring the binary to be present in every default OS installation.
+
+To enable the etcd v3 service, first ensure that `etcd2.service` is disabled.
+
+```
+$ systemctl list-units | grep etcd2.service   # check if etcd2.service is enabled
```

Comment:
`systemctl status etcd2` would suffice, but -
I don't think this doc needs this. A disable and stop is idempotent to current service status, and etd2 service is shipped disabled.